handlers.go

```diff
@@ -3667,10 +3666,15 @@ func (s *Server) handleCreateContent(c echo.Context, u *User) error {
 		}
 	}
 
+	bserv := blockservice.New(s.Node.Blockstore, s.Node.Bitswap)
+	dserv := merkledag.NewDAGService(bserv)
+	contentType := util.FindCIDType(context.Background(), req.Root, dserv)
```

Comment:
Just wondering if we can wrap this into a global util function since the `Blockstore`?

`func util.FindCIDContentType(bs blockstore.Blockstore, rem exchange.Interface, root cid.Cid)  (contentType ContentType)`

Reply:
So the problem here is that sometimes we already have `dserv` but not necessarily `bs` or `bserv` for that matter. What I would love though (but not sure it'd be possible) is to have a magic `getDserv` function that simply finds the right blockstore and returns the correct `dserv`